scripts/build_libpixyusb_swig.bat

```diff
@@ -0,0 +1,47 @@
+@ECHO off
```

Comment:
Is this a Windows batch file?

Reply:
Correct. This is to support the Windows end-users.